gravitee-apim-console-webui/src/management/api/documentation-v4/components/api-documentation-v4-pages-list/api-documentation-v4-pages-list.component.scss

```diff
@@ -56,3 +56,10 @@
 .no-pointer {
   cursor: initial;
 }
+
+:host ::ng-deep .mat-mdc-icon-button.mat-mdc-button-base {
+  --mdc-icon-button-state-layer-size: 15%;
+  width: var(--mdc-icon-button-state-layer-size);
+  height: var(--mdc-icon-button-state-layer-size);
+  padding: 12px;
+}
```

Comment:
what about juste remove this : 
https://github.com/gravitee-io/gravitee-api-management/pull/12639/files#diff-059b5fd2fa85d9a4deb3e4230fc8f94106b0dfe5d8b8de301ceca61ffc9a4965R43 
`gap: 8px;`

Reply:
and remove max-witdh here https://github.com/gravitee-io/gravitee-api-management/blob/41ac96180ea28e5e4db80534e395576414d06ee1/gravitee-apim-console-webui/src/management/api/documentation-v4/components/api-documentation-v4-pages-list/api-documentation-v4-pages-list.component.scss#L47
to add 
`width: 0%;`

This way, the action column always takes up the minimum amount of space on the right.